javascript/diff_match_patch_uncompressed.js

```diff
@@ -68,27 +68,16 @@ var DIFF_EQUAL = 0;
 
 /**
  * Class representing one diff tuple.
- * Attempts to look like a two-element array (which is what this used to be).
+ * Constructor returns an actual two-element array, to allow destructuring
+ * See https://github.com/JackuB/diff-match-patch/issues/14 for details
```

Comment:
we can remove this change now, or remove the old code as well. it should be evident to folks that `Diff` is its own class, and the use of `Symbol.iterator` should be an even more clear indicator that this is array-like.